.vscode/settings.json

```diff
@@ -22,7 +22,8 @@
     ],
     "yaml.format.printWidth": 120,
     "files.associations": {
-        "*.role": "cmake"
+        "*.role": "cmake",
```

Comment:
Remove this change

Reply:
Oh I thought I removed this from the commit whoops